boot/makebootable_test.go

```diff
@@ -769,7 +769,7 @@ version: 5.0
 	case standalone && fromInitrd:
 		err = boot.MakeRunnableStandaloneSystemFromInitrd(model, bootWith, obs)
 	case standalone && !fromInitrd:
-		err = boot.MakeRunnableStandaloneSystem(model, bootWith, obs)
+		err = boot.MakeRunnableStandaloneSystem(model, bootWith, obs, nil)
```

Comment:
can be a follow but we might want a test that runs this with mockUnlocker as well

Reply:
I have added it. I have also squashed all the fixups.